Python/Color_Picker/README.md

```diff
@@ -0,0 +1,17 @@
+# COLOR_PICKER FROM IMAGE
+
+Picking color from image is used in the computer Vision application.Here is the python script which gives the name of the color and it's RGB value when clicked on the image. These scripts uses OpenCV, pandas and argpass.
+Before running the script make sure you have OpenCV installed in system.
+
+To run the script :
+Write "python Color_pick.py -i imgpath" on the comand line.
+ <a href="https://imgur.com/W1dhN0x"><img src="https://i.imgur.com/W1dhN0x.png" title="source: imgur.com" /></a>
+
+image will be shown as soon as you run these command
+
+<a href="https://imgur.com/orZ4YZf"><img src="https://i.imgur.com/orZ4YZf.png" title="source: imgur.com" /></a>
+
+Output on clicking a point on image :
+<a href="https://imgur.com/VA4Y83O"><img src="https://i.imgur.com/VA4Y83O.png" title="source: imgur.com" /></a>
+
+To exit we need to click q
```

Comment:
Can you please follow the standard template of making README [here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md) ?

Reply:
Sure I will re do it